dts/msm8216-samsung-r05.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
```

Comment:
Can you try if

```suggestion
	qcom,msm-id = <248 0>;
```

works? That should be more correct. Thanks.

Reply:
Yes, it works.